### ISSUE TITLE
Source Code

### ISSUE THREAD

**Issue by InjectionDev** [EUOCheffe/easyuo#2]
Am i able to compile a full version of EasyUO or OpenEUO with the files on this repo ?

**Comment by EUOCheffe** [comment]
No, not yet. There are still open parts I need to convert and adapt. The demo code works and can be used in other projects already. But it's no EasyUO yet.
